code-challenges.test.js

```diff
@@ -34,8 +79,35 @@ const letterA = "a"
 const letterE = "e"
 // Expected output: ["Cherry", "Blueberry", "Peach"]
 
+
+describe("arrayFromLetter", ()=>{
+    it("takes in an array of words and a single letter and returns an array of all the words containing that particular letter.", ()=>{
+        expect(arrayFromLetter(letterA)).toEqual(["Mango", "Apricot", "Peach"])
+        expect(arrayFromLetter(letterE)).toEqual(["Cherry", "Blueberry", "Peach"])
+    })
+})
+
 // b) Create the function that makes the test pass.
 
+//Pseudo Code
+//create a function named arrayFromLetter
+//takes in an array of words
+//use .filter higher order function to iterate over the array and find out if it contains the letter 
+//if it does, filter those into an array and return the filtered array.
+
+
+
+const arrayFromLetter = (array, letter) => {
+    return letterArray = array.filter((value) => {
+        return value.toLowerCase.includes(letter)
+    })
+}
+
+arrayFromLetter(letterA)
+arrayFromLetter(letterE)
+
+//I'm now getting the same error, that .filter is not a function. I know i ran into this problem before, and knew how to fix it, but I can't remember the steps to take here. I've always had problems with getting higher order functions to work, so I need to work on them more. 
```

Comment:
Your test is only providing one variable but your function expects two. Looks like your logic and psuedo code are perfectly sound it's just the set up details getting past you. I know that feeling.